src/bootstrap/dist.rs

```diff
@@ -602,6 +602,8 @@ impl Step for DebuggerScripts {
             // gdb debugger scripts
             builder.install(&builder.src.join("src/etc/rust-gdb"), &sysroot.join("bin"),
                     0o755);
+            builder.install(&builder.src.join("src/etc/rust-gdbui"), &sysroot.join("bin"),
```

Comment:
Typo here: it should be `src/etc/rust-gdbgui`, with a `g`

Reply:
Good catch! Fixed in bf9c0fb650e94fc522e1583d557c43320d2f41c0.